projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -171,6 +171,8 @@ public function __construct( $attributes, $content = null ) {
 			'salesforceData'         => null,
 			'hiddenFields'           => null,
 			'stepTransition'         => 'fade-slide', // The transition style for multi-step forms. Options: none, fade, slide, fade-slide
+			'uuid'                   => '', // Unique UUIDv4 identifier for the form
+			'name'                   => '', // Name identifier for the form, not necessarily unique
 		);
```

Comment:
Most of the changes in this PR are for adding the attribute to the editor/block. We just have this code on the frontend. Is this enough to ensure the name and uuid are submitted along with other form data? I assume that's ultimately the purpose is to be able to use these elements on the form submission data.